src/agent/driver-test-runner.js

```diff
@@ -157,14 +157,18 @@ export class DriverTestRunner {
     });
 
     await asyncOperation();
+    // this.log(AgentMessage.DEBUG, {msg: '_collectSpeech - operation completed'});
```

Comment:
more logging that I needed while testing, but could probably stay commented / delete these now